lint/legacy.py

```diff
@@ -0,0 +1,199 @@
+import sublime
+from . import util, scheme
+from xml.etree import ElementTree
+from .persist import settings
+
+import re
+import os
+
+
+COLOR_SCHEME_PREAMBLE = '''<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+'''
+
+
+def touch_dir(dir):
+    """Create dir if it does not exist."""
+    if not os.path.exists(dir):
+        os.makedirs(dir)
+
+
+class XmlScheme(scheme.Scheme):
+    """docstring for XmlScheme"""
+
+    def generate_color_scheme_async(self):
+        """
+            Generate a modified copy of the current color scheme that contains SublimeLinter color entries.
+
+            The current color scheme is checked for SublimeLinter color entries. If any are missing,
+            the scheme is copied, the entries are added, and the color scheme is rewritten to Packages/User/SublimeLinter.
+
+            """
+        print("XmlScheme.generate_color_scheme called.")
```

Comment:
Can we only print these in debug mode?

Reply:
5560c90 Done. Removed all `print` statement I have introduced.